core/src/main/resources/jenkins/slaves/DeprecatedAgentProtocolMonitor/message.properties

```diff
@@ -0,0 +1,4 @@
+blurb=This Jenkins instance uses deprecated Remoting protocols: {0}. \
```

Comment:
`AgentProtocol` is not restricted to Remoting, though I am unaware of any non-Remoting-based protocols other than the trivial ping.